Integrations/integration-cyberark_AIM.yml

```diff
@@ -81,6 +97,59 @@ script:
         return JSON.parse(res.Body);
     }
 
+    var sendRequestWithToken = function(method, url, args, returnPlainResponse) {
+        var headers = {}
+        headers["Content-Type"] = ["application/json"]
+        if (!token) {
+            var data = {
+              username: username,
+              password: password,
+              useRadiusAuthentication: false,
+              connectionNumber: 1
+            };
+            var tokenRes = http(
+                authUrl + 'Logon',
+                {
+                    Method: 'POST',
+                    Headers: headers,
+                    Body: JSON.stringify(data)
+                },
+                insecure,
+                proxy
+            );
+            if (tokenRes.StatusCode !== 200) {
+                throw 'Could not authenticate user ' + tokenRes.Body;
+            }
+
+            var tokenResObject = JSON.parse(tokenRes.Body);
+            token = tokenResObject.CyberArkLogonResult;
+        }
+
+        headers["Authorization"] = [token];
+        if (args.ImmediateChangeByCPM) {
+            headers["ImmediateChangeByCPM"] = [args.ImmediateChangeByCPM];
```

Comment:
it should be in `headers` and in `body`?

Reply:
Ill remove it from the body!